addons/bus/static/src/js/services/bus_service.js

```diff
@@ -1,55 +1,14 @@
-odoo.define('bus.BusService', function (require) {
-"use strict";
+/** @odoo-module **/
```

Comment:
this file should be in `static/src`, not in `static/src/js/services`. The `js` folder is deprecated. Also, for the bus addon, you could simply put all your file top level:
```
assets_watchdog_service.js
bus_service.js
legacy_bus_service.js
legacy_websocket_service.js
websocket_service.js
websocket_bus.js
websocket_errors.js
websocket_worker.js
websocket_worker_...